Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -203,13 +201,13 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         Task {
             await updateBandwidthAnalyzer()
 
+            // This provides a more frequent active user pixel check
+            providerEvents.fire(.userBecameActive)
```

Comment:
There was no reason to do this only when the connection was idle, so instead we'll do it no matter what.